wolox-training/mr-rails
# Localization with i18n

# Summary
* Added i18n gem
* Translated all texts to spanish

# Test Plan
1. Go to /articles
2. Select 'Español' on the dropdown on the top of the page
3. See that the page is now displayed in spanish

# Trello Card
https://trello.com/c/SixmyyqF/10-internacionalizar-el-blog
